c2rust-analyze/tests/analyze.rs

```diff
@@ -0,0 +1,18 @@
+use common::Analyze;
+
+pub mod common;
+
+#[test]
+fn string_literals() {
+    Analyze::new().run("tests/analyze/string_literals.rs");
```

Comment:
could you try making these filecheck tests instead, given the CI failure?

Reply:
Per our discussion, we decided we'd avoid running through `cargo` to fix the CI failure, a simplification.  Then when we need `FileCheck` tests for these, we'll move them to `filecheck/` and add the directives.